src/tools/interop/idt/capture/ecosystem/play_services/play_services.py

```diff
@@ -33,7 +39,8 @@ class PlayServices(EcosystemCapture):
     """
 
     def __init__(self, platform: Android, artifact_dir: str) -> None:
-
+        self.logcat_fd = None
+        self.logger = logger
```

Comment:
why is this needed? Is this expected to be overriden? How about using the top level logger (and call it LOGGER assuming a constant ... although given that it is a mutable object I am unsure)

Reply:
Previously, logcat_fd was opened in analyze and closed on stop, so the file objrect was maintained as an instance var. Now that we instead cleanup by catching the task cancellation, I will remove this. Also TAL at the logger.

Reply:
Done

Reply:
Actually the last commit has an issue where logcat file not getting closed here after that change, temp reopening.

Reply:
Fixed